src/Block/FeatureMediaBlockService.php

```diff
@@ -13,7 +13,7 @@
 
 namespace Sonata\MediaBundle\Block;
 
-use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\BlockBundle\Form\Mapper\FormMapper;
```

Comment:
```suggestion
use Sonata\AdminBundle\Form\Mapper\FormMapper as AdminFormMapper;
use Sonata\BlockBundle\Form\Mapper\FormMapper;
```
Use AdminFormMapper in `buildCreateForm` and `buildEditForm`
Use FormMapper in `configureCreateForm` and `configureEditForm`

This can make some problem, I will check it.